Week-2/1.Flight-booking.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+function getFullName(firstname, surname, useFormalName, gender) {
+  if (firstname.length == 0 || surname.length == 0) {
+    return `Invalid Input`;
```

Comment:
Catch the errors before you continue running the function; `return`-ing prevents the rest of the code to be run